asset_dashboard/views.py

```diff
@@ -64,7 +64,7 @@ class ProjectListView(LoginRequiredMixin, ListView):
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
 
-        # send the form to a modal in this view
+        # send a new Project form to the modal in this view's template
         form = ProjectForm()
```

Comment:
was confused by this at first but then i remembered how this works in the ui. in the project list view (the home page), a new project is added in a modal form. the new project has some very basic details then the user is redirected to the project edit page, where they can fill out more details.